i18n/en.json

```diff
@@ -2664,27 +2680,31 @@
   },
   {
     "id": "api.templates.mfa_activated_body.info",
-    "translation": "Multi-factor authentication has been added to your account on {{ .SiteURL }}.<br>If this change wasn't initiated by you, please contact your system administrator."
+    "translation": "Multi-factor authentication has been added to your account on {{ .SiteURL }}."
   },
   {
     "id": "api.templates.mfa_activated_body.title",
     "translation": "Multi-factor authentication was added"
   },
+  {
+    "id": "api.templates.mfa_change_extra",
```

Comment:
Wondering if we can combine this with `api.templates.mfa_change_extra` so we don't have a duplicate string to translate?

Reply:
Sorry, this is my mistakes.
`api.templates.mfa_change_extra` has never been used anywhere, so it should be removed.